### ISSUE TITLE
ЮОПам все-таки и.?

### ISSUE THREAD

**Issue by solanum-tuberosum** [abaevdict/abaevdict-tei#14]
Встретилось интересное слово (ræftæ), которое в основном дигорское
в итоге в примерах дигорские примеры не помечаются, помечаются иронские
я сделала просто у всех дигорских примеров xml:lang os-x-digor, а для иронских делала convert to Digor, а потом через xml:lang меняла язык на иронский.
Но просто мы в какой-то момент вроде обсуждали, что ЮОПам. это южно-осетинский? и что можно автоматически везде, где ЮОПам проставить ю.? видимо, в общем, нельзя, потому что здесь ЮОПам, но специальная помета и.
![image](https://user-images.githubusercontent.com/13102259/156567775-082025b9-64c3-4e48-a6ff-16857e1ebf59.png)

**Comment by obelyaev** [comment]
Да, и по фонетике тоже получается не югоосетинский… Там же в Южной Осетии тоже разные диалекты, в т.ч. ближе к "северному" иронскому. Так что не знаю, м.б. лучше всё-таки оставлять так, как у Абаева. А про какие случаи мы обсуждали, что ЮОПам. по умолчанию "ю."? Про иронские слова, где ЮОПам. даны как обычные примеры? Или в дигорском такое встретилось, что пример с ЮОПам., но без пометы?
